### PR TITLE
NGMA-1208: Vertical Stepper component

### PR DESCRIPTION
**Pull request by rozen-brighte**
## Description

Added a new vertical stepper component 

## Associated Tickets

https://brighte.atlassian.net/browse/NGMA-1208

## Checklist

- [x] I've updated unit tests where needed
- [x] I've updated the components README.md where needed
- [x] I've added major version bumps for my breaking changes
- [x] I've added changesets where needed

### CODE REVIEW

File: .changeset/thin-llamas-give.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+'@spark-web/vertical-stepper': major
+'@spark-web/heading': minor
+---
+
+Updated heading component to accept muted variant
+
+New vertical stepper component
```
Comment (christopher-mitchell242) [inline .changeset/thin-llamas-give.md:8]: whoop whoop

File: packages/vertical-stepper/README.md
Comment (christopher-mitchell242) [inline packages/vertical-stepper/README.md:1]: nice!

File: packages/vertical-stepper/src/vertical-stepper.tsx [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+  isCompleted,
+  onEdit,
+  completedContent,
+}: StepProps) => {
+  const theme = useTheme();
+  const [completedContentRef, { height: completedContentViewHeight }] =
+    useMeasure();
+  const [activeContentRef, { height: activeContentViewHeight }] = useMeasure();
+
+  const completedContentAnimatedStyle = useSpring({
```
Comment (christopher-mitchell242) [inline packages/vertical-stepper/src/vertical-stepper.tsx:48]: Can we do something in the front-end guild around the animations, im terrible at this stuff and would love to be better at it!

File: packages/vertical-stepper/src/vertical-stepper.test.tsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+    expect(getByRole('heading', { level: 3, name: 'Step 1' })).toBeTruthy();
+    expect(getByRole('heading', { level: 3, name: 'Step 2' })).toBeTruthy();
+  });
+
+  it('displays the active content for the active step', () => {
+    const { getByText } = render(
+      <VerticalStepper steps={steps} activeIndex={1} />
+    );
+    expect(getByText('Step 2 content')).toBeTruthy();
```
Comment (christopher-mitchell242) [inline packages/vertical-stepper/src/vertical-stepper.test.tsx:44]: The default behavior is that, when one section is open, the other sections should show their "heading", is that right?

Assuming so, can we add the assertion:
`expect(getByRole('heading', { level: 3, name: 'Step 1' })).toBeTruthy();`

Comment (christopher-mitchell242) [reply packages/vertical-stepper/src/vertical-stepper.test.tsx:44]: Ah, just read the next test, oops, ignore this!!

File: packages/vertical-stepper/src/vertical-stepper.test.tsx
Comment (christopher-mitchell242) [inline packages/vertical-stepper/src/vertical-stepper.test.tsx:1]: Nice suite of tests!!

**Review by christopher-mitchell242 — APPROVED** [review]

**Review by yaminibrighte — APPROVED** [review]

**Pull request merged**